docs/circleci-recommendation.md

```diff
@@ -1,3 +1,53 @@
 # CircleCI Recommendation
 
-**TODO** If you are a Circle CI user who is willing to help, please see Issue #37
+## Summary
+
+```yml
+- run:
+   command: |
+     vstest.console.exe ... --logger:"junit;LogFilePath=TestResults/test-result.xml;FailureBodyFormat=Verbose"
+- store_test_results:
+   path: TestResults/
+```
+
+## Details
+
+CircleCI uses just a few pieces of the JUnit XML report to generate the displayed user interface.
+<!--
+Maintenance note:
+Findings as of July 20 2023.
+ - Only data in a <testcase> element is shown.
+ - The logger only puts console text in at the <testsuite> level so CircleCI ignores it.
```

Comment:
I'm a bit confused here. It says below system-out and system-err are shown, but this sounds like it isn't shown?

Why don't we consolidate this so its all user visible. Go ahead and be explicit about limitations in the docs.

Reply:
Re: confusion

Right now, the logger takes all the console output from both the setup, closedown and individual tests, turns it into a single string, and places that in a <system-out> element in the test**suite** ... but, at present, CircleCI ignores anything in the test**suite**; CircleCI only shows data from the test**case**, not the test**suite**.

e.g.
```xml
<testsuites>
  <testsuite>
    <testcase>
      <properties>
        <property name="CircleCI" value="Does not show these" />
      </properties>
      <failure message="This scan be seen in CircleCI">So can this</failure>
      <system-out>This would be shown too, but the logger does not output this.<system-out>
      <system-err>I expect the same would apply for system-err too</system-err>
    </testcase>
    <system-out>The logger puts data here but CircleCI ignores it</system-out>
    <system-err>I expect the same would apply for system-err too</system-err>
  </testsuite>
</testsuites>
```


Re: Why
CircleCI is a moving target; I thought it'd be helpful to include some kind of "this was the case at \<date>" text so that the next person to edit this file could better understand why it said what I've written.
... but, while I like to have some kind of comment saying why things have been written the way they were, I don't believe that having this "out in the open" yields the prettiest documentation.

It could be turned into some kind of "footnote" if you'd prefer that kind of commentary visible for all to see.

Reply:
As it needed explanation (IMO, if someone asks questions about documentation then the documentation wasn't clear enough) I've moved the "maintenance note" into a footnote _and_ added a bit of sample XML highlighting what CircleCI does/does-not display.